recipes/default.rb

```diff
@@ -0,0 +1,20 @@
+#
+# Cookbook Name:: netapp_e
+# Recipe:: default
+#
+# Copyright 2016, Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+chef_gem 'excon'
```

Comment:
You're going to want to set this to compile time to have a higher chance of it being available for other resource use.

chef-vault's cookbook does a good job of doing the right thing: https://github.com/chef-cookbooks/chef-vault/blob/master/recipes/default.rb